framework/src/main/java/org/checkerframework/framework/stub/StubGenerator.java

```diff
@@ -367,21 +368,14 @@ private void indent() {
     }
 
     /**
-     * Return a string representation of the list in the form of {@code item1, item2, item3, ...}.
+     * Return a string representation of the list in the form of {@code item1, item2, item3, ...},
+     * without surrounding square brackets as the default representation has.
      *
-     * <p>instead of the default representation, {@code [item1, item2, item3, ...]}
+     * @param lst a list to format
+     * @return a string representation of the list, without surrounding square brackets
      */
     private String formatList(List<?> lst) {
-        StringBuilder sb = new StringBuilder();
-        boolean isFirst = true;
-        for (Object o : lst) {
-            if (!isFirst) {
-                sb.append(", ");
-            }
-            sb.append(o);
-            isFirst = false;
-        }
-        return sb.toString();
+        return UtilPlume.join(", ", lst);
```

Comment:
Why can this one not use `SystemUtil.join`? Should we add a corresponding overload, to have one place to look for such `join` methods, instead of looking for either `SystemUtil` or `UtilPlume`?

Reply:
Yes, it can use SystemUtil.join, but SystemUtil.join delegates to UtilPlume and this avoid the extra step.  I feel that eventually it would be good to remove the duplication and have only one version of the method.

Reply:
https://github.com/typetools/checker-framework/pull/3564 takes a step toward that.  I would like to reduce usage of plume-util in favor of Guava, in the future.